rpc/src/slave.x

```diff
@@ -58,14 +59,16 @@ struct commit_vec_args {
     unsigned hyper int vector<>;
 };
 
+/* TODO: Should TPC RPCs include a vector_clock component in return value? */
```

Comment:
Okay, so we will need to create a `tpc_result` struct that holds the `vector_clock` as well as any other return information (currently just an `int`).

Reply:
I don't know if TPC has anything to do with causal ordering, since it' going to precede every PUT operation, so I don't know if it's actually necessary. I'll look into it though.